plugin.video.catchuptvandmore/addon.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.catchuptvandmore"
+       name="Catch-up TV &amp; More"
+       version="0.1.0"
+       provider-name="SylvainCecchetto">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.beautifulsoup4" version="4.3.1"/>
+    <import addon="script.module.requests" version="2.9.1"/>
+    <import addon="script.module.simplejson" version="3.3.0"/>
```

Comment:
you don't need this one since krypton. Just use standard json: `import json` on all platforms